net-libs/tox/metadata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
```

Comment:
You need to add yourself as `<maintainer/>`, and `<herd>proxy-maintainers</herd>` above it. This will mean you're directly responsible for the ebuild and handling bugs, and proxy-maintainers people will be around to commit stuff for you.